.github/workflows/rust.yml

```diff
@@ -0,0 +1,50 @@
+on: [push, pull_request]
```

Comment:
Do we want this to run for all, or just to master? (Currently set up as all)

Reply:
We want CI workflows for PRs, if that's what you're asking.

Reply:
Sounds good. I was asking if we want to just run this for PRs from a branch to the master branch, or PRs to any branch overall. I think all would do fine.

Reply:
All sounds great!